### PR TITLE
Added material-ui package

### PR DESCRIPTION
**Pull request by shashankcm**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Did you test this feature on all browsers?

Yes, ensured with all the following browsers:

- [x] Chrome
- [x] Firefox
- [x] Safari
- [x] Edge
- [x] Internet Explorer

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,62 @@
 {
-  "name": "client",
-  "version": "0.1.0",
+  "name": "material-dashboard-pro-react",
```
Comment (Satsh24) [inline package.json]: Please update name and version properly..Name: "Job Search Helper" version: 0.0.1, what say?

File: package.json [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
+    "react-chartist": "0.13.3",
+    "react-datetime": "2.16.3",
+    "react-dom": "16.9.0",
+    "react-google-maps": "9.4.5",
+    "react-jvectormap": "0.0.12",
+    "react-router-dom": "5.0.1",
+    "react-scripts": "3.1.0",
+    "react-swipeable-views": "0.13.3",
+    "react-table": "6.10.0",
+    "react-tagsinput": "3.19.0"
```
Comment (Satsh24) [inline package.json]: Do we really need all these packages ??

File: public/index.html [hunk 3]
```diff
@@ -1,43 +1,96 @@
...
-    <title>React App</title>
+          Notice the use of %PUBLIC_URL% in the tags above.
+          It will be replaced with the URL of the `public` folder during the build.
+          Only files inside the `public` folder can be referenced from the HTML.
+
+          Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
+          work correctly both with client-side routing and a non-root public URL.
+          Learn how to configure a non-root public URL by running `npm run build`.
+        -->
+    <title>Material Dashboard React by Creative Tim</title>
```
Comment (Satsh24) [inline public/index.html]: update name..

File: package.json (hunk 1 above)
Comment (shashankcm) [reply package.json]: the name should not have space and capital letters.  so i change it to "name": "job-search-helper",

Comment (Satsh24) [reply package.json]: version ??

**Pull request merged**